src/dbos-executor.ts

```diff
@@ -928,6 +932,13 @@ export class DBOSExecutor implements DBOSExecutorContext {
     return handlerArray;
   }
 
+  async deactivateConsumers() {
+    this.logger.info("Deactivating event receivers");
+    for (const evtRcvr of this.eventReceivers || []) {
+      await evtRcvr.destroy();
+    }
+  }
```

Comment:
Does this also shut down the queue polling loop, or does that need to be done elsewhere?

Reply:
I assume you are referring to the WF Queue runner. Will add the code to shut that down as well